addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/UsernameEnumerationScanRule.java

```diff
@@ -602,8 +602,7 @@ public void scan() {
                 // if we didn't hit something with one of the iterations for the parameter (ie, if
                 // the output when changing the param is stable),
                 // check if the parameter might be vulnerable by comparing its LCS with the
-                // original
-                // LCS for a valid login
+                // original LCS for a valid login
```

Comment:
This is really unnecessary.

Reply:
Yes it’s only a whitespace change but it does make it read easier